csync/src/std/asprintf.c

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Should the file go in 3rdparty?
Should there be a URL from the readme on where this came from, and which revision. ?